### ISSUE TITLE
Settings: learn more element should be combined into a single focusable element

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-Android#10940]
Steps to reproduce:

1. Go to Settings > Accessibility > TalkBack > "Use service" or use the [Volume key shortcut](https://support.google.com/accessibility/android/answer/6006966?hl=en).
1. In the WordPress app, go to My Sites > Settings > More.
1. Swipe right to find the "Learn More" element.

**Result:** ([48s](https://cloudup.com/cLRyGISEorB))

> This part of the Settings – More screen contains two elements that are focusable for TalkBack: (1) the text field, and (2) the learn more button. TalkBack reads the text of the first element (“You can…”) and finishes with “double tap to activate, double tap and hold to long press.” Double-tapping does nothing, however. Double tapping and holding opens a popup dialog. If the learn more button is focused, double tapping opens the same dialog. This could be improved by combining these two focusable elements into a single focusable element that opens the relevant dialog if activated.

<img src="https://user-images.githubusercontent.com/1119271/70570063-6c85c700-1b58-11ea-9ad8-bff2e94ba5c4.png" width="270" alt="learn-more-element">
<sup>Tested with WPAndroid 13.8-rc-1 on Pixel 3 Android 10.</sup>

<p style="clear:both;"><sub>/hat tip @mchowning for the report, internal reference: p77Llu-c0a-p2</sub></p>

**Comment by nikopetr** [comment]
Hello , can you explain a bit more in detail what exactly you need to be shown instead ?

**Comment by nikopetr** [comment]
Hello is this still up ?

**Comment by shiki** [comment]
@jd-alexander Are you still working on this? If not, maybe @nikopetr can take a look?

**Comment by jd-alexander** [comment]
@shiki Yes, I am going to do look at this one and the other.

**Comment by A-L-Fearon** [comment]
Howdy @jd-alexander, is this issue still available ?

**Comment by jd-alexander** [comment]
@A-L-Fearon yes it is.

**Comment by Bawya1098** [comment]
@jd-alexander. Hello, I would like to try and Contribute and work on this. 
Could you please help me with the authentication part?

**Comment by jd-alexander** [comment]
Hi @Bawya1098 Thanks for volunteering to contribute. We really appreciate it 😄 

> Could you please help me with the authentication part?

Could you expound a bit on the issues you are having? Thank you.

**Comment by A-L-Fearon** [comment]
@jd-alexander thanks for the response, was inquiring to take this on but @Bawya1098 beat me to it 😅

**Comment by Bawya1098** [comment]
@jd-alexander, I was not  clear with the ` OAuth2 Authentication`, 
I'm able to build it locally but not able to login.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
